native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/serializer/GlueSchemaRegistryKafkaSerializer.cs

```diff
@@ -0,0 +1,44 @@
+
+using System;
+using AWSGsrSerDe.common;
+
+namespace AWSGsrSerDe.serializer
+{
+    public class GlueSchemaRegistryKafkaSerializer
+    {
+        private readonly IDataFormatSerializer _serializer;
+        private readonly string _dataFormat;
+        private readonly Func<string, object, string> _schemaNamingStrategy;
```

Comment:
Good to extract this into a separate interface to let users easily extend it.

Reply:
makes sense